src/reason_lexer.mll

```diff
@@ -446,6 +446,8 @@ rule token = parse
         lexbuf.lex_curr_p <- { curpos with pos_cnum = curpos.pos_cnum - 1 };
         STAR
       }
+  | "#" (appropriate_operator_suffix_chars | "#")
```

Comment:
I was actually just cleaning this up to address my previous issues with it and added a "+", which I believe is more appropriate?